src/shared-components/ActionItems/ActionItemModal/ActionItemModal.autocomplete-coverage.spec.tsx

```
@@ -0,0 +1,246 @@
+  result: unknown;
+};
+
+// Mock MUI Autocomplete to deterministically invoke `isOptionEqualToValue`.
+// This avoids flaky Popper/listbox rendering in jsdom while still executing
+// the inline lambdas inside `ActionItemModal.tsx` for coverage.
+vi.mock('@mui/material/Autocomplete', async () => {
+  const React = await import('react');
+
+  const __autocompleteCalls: AutocompleteCall[] = [];
+
+  const Autocomplete = (props: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Insert `⏎·······`

(prettier/prettier)